pulsar-broker/src/main/java/org/apache/pulsar/broker/lookup/v2/TopicLookup.java

```diff
@@ -45,6 +49,10 @@ public class TopicLookup extends TopicLookupBase {
     @GET
     @Path("{topic-domain}/{tenant}/{namespace}/{topic}")
     @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(
+            value = "Look up a topic from the current serving broker.",
```

Comment:
Should it be `Get the owner broker of the given topic`?

Reply:
Agree,  Maybe also need to change it?

https://github.com/apache/pulsar/blob/81841eba724e41d243ba7afc58fcc7634513938b/site2/docs/reference-pulsar-admin.md?plain=1#L2653-L2654

Reply:
This doc page is deprecated and no longer maintained.
You can find the related information on [this page](https://pulsar.apache.org/docs/next/admin-api-topics/#lookup-of-topic).